index.html

```diff
@@ -36,250 +37,356 @@
+                </div>
+                <div class="btn-container">
+                  <button type="button" class="btn">
+                    <a href="#">See Project</a>
+                  </button>
+                </div>
+              </div>
+            </div>
+          </article>
         </li>
       </ul>
     </section>
+    <!-- ABOUT SECTION -->
     <section id="about-container">
-      <ul id="about">
-        <li>
-          <h2 class="header">
-            <span>About<br />Myself</span>
-          </h2>
-        </li>
-        <li>
-          <p class="text">
-            I’m a software developer! I can help you build a product, feature or
-            website. Look through some of my work and experience! If you like
-            what you see and have a project you need coded, don’t hestiate to
-            contact me.
-          </p>
-        </li>
-        <li>
-          <h2 id="connect">Let's Connect</h2>
-        </li>
-        <li id="social-icons-container">
-          <ul id="social-icons">
-            <li class="icon">
-              <a
-                href="https://github.com/hafedefheij"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                <i class="fa-brands fa-github"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a
-                href="https://www.linkedin.com/in/hafedefheij/"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                <i class="fa-brands fa-linkedin-in"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a href="#" target="_blank" rel="noopener noreferrer">
-                <i class="fa-brands fa-angellist"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a href="#" target="_blank" rel="noopener noreferrer">
-                <i class="fa-brands fa-twitter"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a href="#" target="_blank" rel="noopener noreferrer">
-                <i class="fa-brands fa-medium"></i>
-              </a>
-            </li>
-          </ul>
-        </li>
-        <li>
-          <div id="button-container">
-            <button type="button" class="btn">
-              <a href="#">Get my resume</a>
-            </button>
+      <article id="about">
+        <div id="description-container">
+          <div id="description">
+            <h2 class="header">
+              <span>About Myself</span>
+            </h2>
+            <p class="text" id="about-text">
+              I’m a software developer! I can help you build a product, feature
+              or website. Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <h2 id="connect">Let's Connect</h2>
```

Comment:
- [ ] Awesome job, @hafedEfheij :clap: The responsiveness is iconic now, with better user experience. However, as the previous reviewer provided guides on the images he left for you, the `font` properties are not as required at the moment. Visibility of your page should be very much accessible to users, so that the details you need them to see or know in order to guide them in taking actions will be clear to nearly all types of users. For more clarity, check the image below:

![image](https://user-images.githubusercontent.com/99715640/236638448-b678e733-171a-47e5-b1b6-ec57b6659fea.png)